### ISSUE TITLE
Stuck on "Checking to see if wallet file is encrypted" Page

### ISSUE THREAD

**Issue by Bigcow11** [project-ecc/Sapphire#26]
You are stuck on the "Checking to see if wallet file is encrypted" page after importing a wallet.dat file.
Happening on the initial page of "Setting a few things up for you" as well.
Mac OS 10.13.4
Sapphire 1.0.1

**Comment by csmartinsfct** [comment]
You can't be stuck on one panel and on another different panel simultaneously. Please detail your case and the steps you took to become stuck. Make sure you are using the latest version of the google drive link. It was uploaded today.

https://drive.google.com/drive/folders/1PL9RRzseJnt4v82vEAszB3bCgWE_RanM?usp=sharing

And make sure you read the guide that is in that link.

**Comment by Bigcow11** [comment]
Sorry about the lack of clarity. So what happened was this, I followed the guide and downloaded the newest version and it initially worked all the way to the "Checking to see if wallet file is encrypted" page then froze on this. I noticed the google folder was just uploaded today like you said so I will try that out and report in the support channel on Slack. Thanks but if I have some issues after retrying I will post it in the Slack.
